tests/_files/NumericGroupAnnotationTest.php

```diff
@@ -7,11 +7,17 @@
  * For the full copyright and license information, please view the LICENSE
  * file that was distributed with this source code.
  */
+
+/**
+ * @author Companion Cube
+ * @ticket t123456
+ */
 class NumericGroupAnnotationTest extends \PHPUnit\Framework\TestCase
 {
     /**
      * @testdox Empty test for @ticket numeric annotation values
      * @ticket  3502
+     * @author C. Lippy
```

Comment:
😆

Reply:
📎  It looks like you're writing a PHP test framework. Would you like help?